docs/networking/clusternetwork.md

```diff
@@ -44,14 +44,14 @@ Please refer to [network part](./harvester-network.md) for more details about ne
 
 ### Built-in Cluster Network
 
-Harvester provides a built-in cluster network called `mgmt`. It's different from the custom cluster network. `mgmt`:
+Harvester provides a built-in cluster network called `mgmt`. It's different from the custom cluster network. The mgmt cluster network:
 
 - Cannot be deleted.
 - Does not need any network configuration.
 - Is enabled on all hosts and cannot be disabled.
 - Shares the same traffic egress with the management network.
 
-If there is no need for traffic separation, you can put all your network under the `mgmt` cluster network.
+If there is no need for traffic separation, you can put all your network under the mgmt cluster network.
 
```

Comment:
`If there is no need for traffic separation` -> `When traffic isolation is not required`

btw, use `traffic separation` or `traffic isolation` or `network isolation` ?  @vickyhella  your idea?

Reply:
I usually use `traffic isolation` and `separate traffic`.